### DISCUSSION TITLE
Welcome to nexmon Discussions!

### DISCUSSION THREAD

**Discussion by matthiasseemoo** [seemoo-lab/nexmon#467] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by internrayiot** [comment]
Is it possible to perfrom CSI data extraction using nexmon tool happens in the following phones with the respective firmware patches given below and how?
Samsung Galaxy S22 Plus - 20_82_42_sta (r994653)
Google Pixel 7 and 7 Pro - 20_101_36_2 (r994653) and 20_101_57 (r1035009)
Google Pixel 8 - 24_671_6_9 (r1031525)

**Reply by jlinktu** [reply]
Yes, we are currently working on that.

**Reply by internrayiot** [reply]
Apart from these phones and also, Nexus 5 and Nexus 6P, which other phones support extraction of CSI data using Nexmon tool.

**Reply by jlinktu** [reply]
Our current extractor that can be found at https://github.com/seemoo-lab/nexmon_csi only support the Nexus 5 and Nexus 6P. In theory, this can be done on any device that features a BCM43-family Wi-Fi SoC.

**Reply by internrayiot** [reply]
So, can the same steps that are used to extract CSI data from Nexus 5 and Nexus 6P, be used to extract the CSI data from any device with BCM43-family Wi-Fi SoC.

**Reply by jlinktu** [reply]
At least similarly.
We will publish a remake of our extractor not too far in the future that supports also newer chipsets, i.a. those you are asking for.

**Reply by internrayiot** [reply]
How long will take to release the firmware to extract the CSI data using these devices?

**Reply by jlinktu** [reply]
There is no fixed date. We are done when we are done.

**Comment by internrayiot** [comment]
Hi I was looking for Samsung S20 firmware patches for Extraction of CSI data, for BCM4375b1 WiFi chipset.

Let me know if its available, since I could not find any patches.

**Reply by jlinktu** [reply]
I already gave you an answer to this: https://github.com/seemoo-lab/nexmon/discussions/467#discussioncomment-11457003
